.github/workflows/maven-ci.yml

```diff
@@ -24,24 +24,29 @@ jobs:
     continue-on-error: ${{ matrix.experimental }}
 
     steps:
-      - name: Checkout repo
+      - name: Checkout
         uses: actions/checkout@v4
 
-      - name: Configure Java & Maven
-        uses: actions/setup-java@v4
-        with:
-          java-version: '${{ matrix.java-version }}'
-          cache: "maven"
-          distribution: "corretto"
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
 
-      - name: Build with Maven
-        run: mvn -B -e -ntp "-Dstyle.color=always" -Dmaven.compiler.release=${{ matrix.java-compiler }} verify
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
 
-      - name: Verify Changed Files
```

Comment:
Not sure if we care. But now the only step that checks for changed files from the formatter is the mac build.

Reply:
I originally had it as a step in the Dockerfile to check, but I don't want it to fail if you're doing local development and testing.

Reply:
Separate runner??

Reply:
Feels a little overkill. But runners are free. So sure. Unless you think it is too much.

Reply:
Feels a little overkill, but sorta nice to have it as a separate runner. We could probably add things to it in the future if we wanted to? @jpdahlke Thoughts?

Reply:
We've had PRs just to correct a branch that got merged that produces formatting changes.  Personally, I wish the build would fail sooner before the branch gets merged or even the PR gets created. But as long as this will make sure the changes get addressed before merge than it's probably fine to have it separated out.

Reply:
@ldhardy 
Here in emissary those formatting changes have blocked the merge since the CI check fails. Don't think we have needed those follow prs recently here. 

But yeah good point.